docs/user-guide/knowledge-distillation.rst

```diff
@@ -0,0 +1,362 @@
+#############################
+
+Once the data has been prepared, you are ready to fine-tune the student model.  This step takes around 50 minutes on 8 H100 80G GPUs.
+
+.. tab-set::
+
+    .. tab-item:: Terminal
+        :sync: key3
+
+         To run fune-tuning on the terminal directly, use the following command. For successful execution, ensure that the NeMo-Aligner repository is set as your current working directory.
+
+         .. code-block:: bash 
+
+            python -u examples/nlp/gpt/train_gpt_knowledge_distillation.py \
+               trainer.num_nodes=1 \
+               trainer.devices=8 \
+               trainer.precision=bf16 \
+               trainer.knowledge_distillation.limit_val_batches=5 \
+               trainer.knowledge_distillation.max_steps=100 \
+               trainer.knowledge_distillation.val_check_interval=1000 \
+               trainer.knowledge_distillation.save_interval=1000 \
+               pretrained_checkpoint.restore_from_path=2b_student.nemo \
+               model.megatron_amp_O2=True \
+               ++model.tensor_model_parallel_size=1 \
+               ++model.pipeline_model_parallel_size=1 \
+               model.micro_batch_size=1 \
+               model.global_batch_size=128 \
+               model.optim.lr=1e-5 \
+               model.optim.name=distributed_fused_adam \
+               model.optim.weight_decay=0.01 \
+               model.optim.sched.constant_steps=0 \
+               model.knowledge_distillation.target_logits_scale=1.0 \
+               model.knowledge_distillation.logits_scale=1.0 \
+               model.knowledge_distillation.sft_loss_weight=0.4 \
+               model.knowledge_distillation.kd_loss_weight=1 \
+               model.knowledge_distillation.kd_loss=fwd_kl \
+               "model.data.data_prefix={train: [data/oasst/train_with_logits_CHUNK_ID.jsonl], validation: [data/oasst/val_with_logits_CHUNK_ID.jsonl], test: [data/oasst/val_with_logits_CHUNK_ID.jsonl]}" \
+               ++model.data.data_impl=chunked_jsonl \
+               ++model.data.n_chunks=1 \
+               ++model.data.n_examples_per_chunk={train: 56440, validation: 2938, test: 2938}" \
+               ++model.data.seq_length=4096 \
+               model.data.splits_string=\'98,1,1\' \
+               exp_manager.create_wandb_logger=True \
+               exp_manager.wandb_logger_kwargs.name=sft_knowledge_distillation_70b_chat \
+               exp_manager.wandb_logger_kwargs.project=sft_knowledge_distillation \
+               exp_manager.explicit_log_dir=results/kd_log_dir \
+               exp_manager.resume_if_exists=True \
+               exp_manager.resume_ignore_no_checkpoint=True \
+               exp_manager.checkpoint_callback_params.monitor=val_loss \
+               exp_manager.checkpoint_callback_params.save_nemo_on_train_end=True
+
+
+    .. tab-item:: Slurm
+        :sync: key4
+
+         To generate the teacher logits via slurm, run the following command: 
```

Comment:
fix spelling.

 To generate the teacher logits via Slurm, run the following command: